components/monitoring/prometheus/base/monitoringstack/monitoringstack.yaml

```diff
@@ -86,6 +87,12 @@ spec:
   endpoints:
   - params:
       'match[]': # scrape only required metrics from in-cluster prometheus
+        - '{__name__="pipelinerun_duration_scheduled_seconds_sum[", namespace="openshift-pipelines"}'
+        - '{__name__="pipelinerun_duration_scheduled_seconds_count[", namespace="openshift-pipelines"}'
+        - '{__name__="pipelinerun_gap_between_taskruns_milliseconds_sum[", namespace="openshift-pipelines"}'
+        - '{__name__="pipelinerun_gap_between_taskruns_milliseconds_count[", namespace="openshift-pipelines"}'
+        - '{__name__="tekton_pipelines_controller_pipelinerun_duration_seconds_sum[", namespace="openshift-pipelines"}'
+        - '{__name__="tekton_pipelines_controller_pipelinerun_duration_seconds_count[", namespace="openshift-pipelines"}'
```

Comment:
Is the `[` at the end of each metric name a typo?

Reply:
yep thanks

Reply:
updated / fixed @amisstea